src/unread/UnreadCards.js

```diff
@@ -59,6 +61,22 @@ export default function UnreadCards(props: Props): Node {
     ...unreadStreamsAndTopics,
   ];
 
+  // move the collpased state of the stream here in a Immutable Set
+  const [uncollapsedStreamIds, setUncollapsedStreamIds] = useState(new Immutable.Set<number>());
```

Comment:
This comment doesn't really express any information that isn't in the line of code that follows it. In that situation, best to leave the comment out and let the code speak for itself.